components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -244,6 +248,7 @@ export const AudioPlayer = ({
               component="a"
               href={url}
               download={audioDownloadFilename}
+              aria-label="Download"
```

Comment:
This should be "Download audio" to be descriptive of what is being downloaded.

Reply:
Fixed!